### DISCUSSION TITLE
Wireshark Warning: Unknown NdisMedium value 12, defaulting to DLT_EN10MB

### DISCUSSION THREAD

**Discussion by rpuskarcik** [community/community#53520] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Select Feature Area

- [X] Issues
- [ ] Projects

### Body

As soon as I attempt to capture packets I get a popup with the following error:
Warning: Unknown NdisMedium value 12, defaulting to DLT_EN10MB
Then nothing... no output whatsoever.

I tried uninstalling npcap and reinstalling, but I can't reinstall because I get another error which says it is still running.

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by queenofcorgis** [comment]
Hi @rpuskarcik ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this WireShark, like their [forum](https://ask.wireshark.org/questions/). It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!

**Comment by cleidson-cmd** [comment]
Opa! boa. eu consegui resolver aqui: desinstalei marquei todas as opções durante a remoção e em seguida instalei novamente e marcando 2 opções:
install NCAP
